internals/compiler.rst

```diff
@@ -524,140 +524,112 @@ statement in ``_PyEval_EvalFrameDefault()``.
+     when parsing source code.
 
-    pegen.c
-        Contains helper functions which are used by functions in
-        :cpy-file:`Parser/parser.c` to construct the AST.  Also contains helper
-        functions which help raise better error messages when parsing source
-        code.
+   * :cpy-file:`Parser/pegen.h`: Header file for the corresponding
+     :cpy-file:`Parser/pegen.c`. Also contains definitions of the ``Parser``
+     and ``Token`` structs.
 
-    pegen.h
-        Header file for the corresponding :cpy-file:`Parser/pegen.c`. Also contains
-        definitions of the ``Parser`` and ``Token`` structs.
+* :cpy-file:`Python/`
+   * :cpy-file:`Python/Python-ast.c`: Creates C structs corresponding to
+     the ASDL types.  Also contains code for marshalling AST nodes (core
+     ASDL types have marshalling code in :cpy-file:`Python/asdl.c`).
+     "File automatically generated by :cpy-file:`Parser/asdl_c.py`".
+     This file must be committed separately after every grammar change
+     is committed since the ``__version__`` value is set to the latest
+     grammar change revision number.
 
-+ Python/
+   * :cpy-file:`Python/asdl.c`: Contains code to handle the ASDL sequence type.
+     Also has code to handle marshalling the core ASDL types, such as number
+     and identifier.  Used by :cpy-file:`Python/Python-ast.c` for marshalling
+     AST nodes.
 
-    Python-ast.c
-        Creates C structs corresponding to the ASDL types.  Also
-        contains code for marshalling AST nodes (core ASDL types have
-        marshalling code in :cpy-file:`Python/asdl.c`).  "File automatically
-        generated by :cpy-file:`Parser/asdl_c.py`".  This file must be
-        committed separately after every grammar change is committed since
-        the ``__version__`` value is set to the latest grammar change
-        revision number.
+   * :cpy-file:`Python/ast.c`: Used for validating the AST.
 
-    asdl.c
-        Contains code to handle the ASDL sequence type.  Also has code
-        to handle marshalling the core ASDL types, such as number and
-        identifier.  Used by :cpy-file:`Python/Python-ast.c` for marshalling
-        AST nodes.
+   * :cpy-file:`Python/ast_opt.c`: Optimizes the AST.
 
-    ast.c
-        Used for validating the AST.
+   * :cpy-file:`Python/ast_unparse.c`: Converts the AST expression node
+     back into a string (for string annotations).
 
-    ast_opt.c
-        Optimizes the AST.
+   * :cpy-file:`Python/ceval.c`: Executes byte code (aka, eval loop).
 
-    ast_unparse.c
-        Converts the AST expression node back into a string
-        (for string annotations).
+   * :cpy-file:`Python/compile.c`: Emits bytecode based on the AST.
 
-    ceval.c
-        Executes byte code (aka, eval loop).
+   * :cpy-file:`Python/symtable.c`: Generates a symbol table from AST.
 
-    compile.c
-        Emits bytecode based on the AST.
+   * :cpy-file:`Python/peephole.c`: Optimizes the bytecode.
```

Comment:
https://github.com/python/cpython/blob/main/Python/peephole.c

[This file](https://github.com/python/cpython/blob/ba18c0b13ba3c08077ea3db6658328523823a33f/Python%2Fpeephole.c) was deleted ([on Jul 30, 2020](https://github.com/python/cpython/commit/6e8128f02e1d36e38e5866f9dc36e051caa47bc9)) - [Commit history](https://github.com/python/cpython/commits/main/Python/peephole.c).